lib/App/DuckPAN/Cmd/Test.pm

```diff
@@ -6,27 +6,44 @@ with qw( App::DuckPAN::Cmd );
 
 use MooX::Options protect_argv => 0;
 
+use Path::Tiny;
+
 option full => (
-	is          => 'ro',
-	lazy        => 1,
-	short       => 'f',
-	default     => sub { 0 },
-	doc         => 'run full test suite via dzil',
```

Comment:
We use tabs because that's what this was all written in initially, several years ago. Same goes for duckduckgo/duckduckgo. All the IA repos were too, but as new devs and staff came in it all got mixed up.

I fought for tabs, but Spaces won the style wars and we decided to put it into the styleguide so the IA code would be consistent.

DuckPAN is mostly in maintenance mode now, and doesn't get much activity, other than changes by me or @zachthompson so I think it's best to leave it with tabs.